lib/orderbook/errors.ts

```diff
@@ -75,6 +76,10 @@ const errors = {
     message: `order with local id ${localId} has a quantity of ${holdQuantity} satoshis on hold, try again later`,
     code: errorCodes.QUANTITY_DOES_NOT_MATCH,
   }),
+  PAIR_CURRENCIES_CANT_BE_SAME: (baseCurrency: string, quoteCurrency: string) => ({
```

Comment:
Not sure about the naming of this error. At the same time I don't have a better alternative. Thoughts @sangaman?

Reply:
Yeah it's a bit clunky, how about `DUPLICATE_PAIR_CURRENCIES`?